### DISCUSSION TITLE
2024-05-17 | Actividad - GitHub OpenSource CDMX & Fedora México

### DISCUSSION THREAD

**Discussion by ManuOSMx** [gittogethers/cdmx#8] [Activities]
# GitHub OpenSource CDMX & Fedora México
_24/05/17_
_Dinámica_

**¡Hola Comunidad!**
Que gusto verte por aquí 👋🏼 

Estamos preparando este entorno en **GitHub** en donde ustedes podrán interactuar mas con la comunidad, compartir sus ideas, ser parte de las discusiones, votar, participar en las actividades y conectar con mas personas.

Y para impulsarte a conocer mas sobre OpenSource con las herramientas de GitHub, hemos preparado esta dinámica para que puedas ganar un voucher de **GitHub Foundations**
## Instrucciones
- ¡Tómate una foto en el evento del 17 de mayo del 2024  y súbela a tu red social favorita taggeando a [@githubcdmx](https://www.instagram.com/githubcdmx/) ! (Puede ser un post o una historia) ❤️
- Agrega un comentario en este discussion platicándonos por qué quieres ganar la certificación, añade tu fotografía junto con el link a la publicación y estarás participando en el sorteo. :octocat:
## Ponderación
- 1 Voto: 1 punto.
- Foto de algún evento de GitHub GitTogethers CDMX: 5 puntos.

> [!IMPORTANT]  
> Tienes hasta el viernes 17 de mayo del 2024 a las 8:00 p.m para compartir tu comentario con tu fotografía.

## Premios

- 3 Vouchers para presentar el examen de Certificación de GitHub Foundations

## Moderadores
- @aleepsy 
- @ManuOSMx

**Comment by srperf** [comment]
![IMG_20240517_185151_492.jpg](https://github.com/gittogethers/cdmx/assets/54183040/5a2adfb1-fedb-488a-b134-a1e84067dac6)

![IMG_20240517_185151_606.jpg](https://github.com/gittogethers/cdmx/assets/54183040/b9dcb028-a252-477d-977d-3dfa47360f18)

https://www.instagram.com/p/C7FyHsQAr8G/?igsh=MXNjMmxlNndxaTVhNQ==

https://twitter.com/SrPerf/status/1791633546013979027?t=oprC5FBSO3ZwcdIFRQtQ8Q&s=19

Reportando desde el evento, con muchas ganas de la certificación para pulir habilidades Githuberas!
@ManuOSMx

**Comment by SolDavidCloud** [comment]
¡Me van a hacer instalar Fedora!
![IMG20240517195707](https://github.com/gittogethers/cdmx/assets/33080295/2338f8b6-b0ed-4d31-a1dd-efe781293309)

**Comment by OscarSantosMu** [comment]
![IMG_20240517_195623_365](https://github.com/gittogethers/cdmx/assets/38818848/432f855d-b970-46e6-a4de-ae05698021f0)

https://www.instagram.com/stories/oscarsantosmu/3370345853913629263?utm_source=ig_story_item_share&igsh=eXgxMmNwaHl0cXE4

**Comment by callmethesamm** [comment]
![b078b274-8e84-4258-bbe1-19228c0dc83b](https://github.com/gittogethers/cdmx/assets/111318957/237771a3-fe77-4a49-b532-1bb6ba2fff98)

**Reply by callmethesamm** [reply]
Bien feli!!!!

**Reply by callmethesamm** [reply]
Para crecer en conocimientos

**Comment by epinedal1900** [comment]
![IMG_20240517_201750_1](https://github.com/gittogethers/cdmx/assets/75105982/312f7fe3-8271-4e84-b726-ba893bc21bf5)

**Comment by saulrodriguezsk** [comment]
![IMG20240517210924.jpg](https://github.com/gittogethers/cdmx/assets/138558346/a1df5b87-7aca-43fd-b487-f86fe833183b)

Tqm Github🫶🏻

**Comment by educast21** [comment]
![Screenshot_2024-05-17-21-10-56-133_com.instagram.android.jpg](https://github.com/gittogethers/cdmx/assets/118428508/878beee7-af7b-4fd1-99fa-1d3a62b7ddce)

Quisiera ganar la certificación para dar mis primeros pasos y seguir aprendiendo 🙆🏻‍♂️

https://www.instagram.com/stories/_educast_/3370389859107687582?igsh=YTQwajdzMnpteGV2

**Comment by joseespiritu** [comment]
![20240517_195514](https://github.com/gittogethers/cdmx/assets/19401002/a105197e-2ddb-4f37-8894-ca6312f353b9)

Github x Fedora 2024 🥸

**Comment by untalbry** [comment]
![Screenshot_20240517_212746_com.instagram.android.jpg](https://github.com/gittogethers/cdmx/assets/91161604/baf54db5-4f58-478d-8a06-6ebf0730dc17)

Porque quiero ganar la cartificacion?
Git y github es una de las herramientas que como dev y uso en mi día a día.
Y es que quien no los usa?

Me encantaría demostrás mis habilidades que tengo en estas tecnologías que todos amamos y que muestra que el open source es grande y tiene una muy bonita comunidad ❤️

**Comment by Hu3rgo99** [comment]
![Uploading IMG_1133.png…]()
Me encantaría aprender mucho más y desarrollarme con la certificación
